samples/openthread/coap_client/prj_nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -25,7 +25,29 @@ CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETS_POSIX_NAMES=y
 CONFIG_NET_SOCKETS_POLL_MAX=4
 
+# Same network Master Key for client and server
+CONFIG_OPENTHREAD_MASTERKEY="00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff"
+
 # Default PRNG entropy for nRF53 Series devices is CSPRNG CC312
 # which for that purpose is too slow yet
 # Use Xoroshiro128+ as PRNG
 CONFIG_XOROSHIRO_RANDOM_GENERATOR=y
+
+# Enable Bluetooth LE controller which supports multiprotocol
```

Comment:
Why enabling multiprotocol by default? Shouldn't we be able to use `overlay-multiprotocol_ble.conf` when desired? Same as for nRF52.

Reply:
You are right, we don't need extra configuration here.